### ISSUE TITLE
acmart documentclass is incompatible with biblatex

### ISSUE THREAD

**Issue by fmetzger** [LateXTemplateModernizationProject/ABOUT#1]
The document class `acmart` (available e.g. from [CTAN](https://www.ctan.org/pkg/acmart)), which is used by many ACM conferences and proceedings, forces the use of natbib. In turn, natbib is incompatible to biblatex.

So, we should produce a variant of acmart without natbib, but with biblatex preconfigured in such a way that it produces the same citation style.

**Comment by fmetzger** [comment]
Other minor quirks with this class:
- Packages `url` and `hyperref` are preloaded, so their options cannot be easily customized in the actual document.

**Comment by aaaaalbert** [comment]
For the biblatex issue, see also https://github.com/borisveytsman/acmart/issues/114 ("Can't use with biblatex") and [the proposed fix](https://github.com/borisveytsman/acmart/commit/e5ff428f25c7e092db814613fdc34323753f0487).

**Comment by fmetzger** [comment]
Good find.
Maybe we should just start setting up a list with the problematic templates and provisional fixes (if any).

**Comment by Blaisorblade** [comment]
> Packages url and hyperref are preloaded, so their options cannot be easily customized in the actual document.

Isn't `\PassOptionsToPackage` a sufficient fix for that? The class docs IIRC only mention the unsupported `acm-preload-hook.tex` trick, so the docs need patching.

**Comment by fmetzger** [comment]
Not necessarily. E.g. it could still result in your passed options being overwritten. Additionally, especially `hyperref `is extremely finicky in its load order, it has to be loaded after some, but before other, packages in order to produce the correct results.

Unfortunately, we haven't looked at `acmart` for some time, and it has seen quite a lot of updates in the meantime. So we would need to take a look at the latest version again anyways.
